statsdaemon.go

```diff
@@ -289,12 +289,12 @@ func processGauges(buffer *bytes.Buffer, now int64) int64 {
 
 		switch {
 		case hasChanged:
-			fmt.Fprintf(buffer, "%s %d %d\n", bucket, currentValue, now)
+			fmt.Fprintf(buffer, "%s %.0f %d\n", bucket, currentValue, now)
```

Comment:
ugh, the documentation never even touched on that: http://graphite.readthedocs.org/en/0.9.12/feeding-carbon.html#the-plaintext-protocol

had to dig for a bit, but it looks like carbon parses it that way as well:
https://github.com/tmm1/graphite/blob/master/carbon/lib/carbon/protocols.py#L66